### PR TITLE
writing first unit test for geterrors

### PR DESCRIPTION
**Pull request by eabarnes1010**
please look over my changes.

### CODE REVIEW

File: bivariate_normal/test_prediction_metrics.ipynb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
```
Comment (RandalJBarnes) [inline bivariate_normal/test_prediction_metrics.ipynb]: <p>Why not wrap this in __name__?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eabarnes1010/hurricane_track/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

Comment (eabarnes1010) [reply bivariate_normal/test_prediction_metrics.ipynb]: <p>I think this might be pointless since it is a notebook.</p>

**Comment by RandalJBarnes** [comment]
I think this should be a standalone .py test file.

File: bivariate_normal/test_prediction_metrics.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    y_data = np.ones((100,5))
+    y_data[:,-1] = 0.
+    onehot_data = 3.*np.ones((100,2))
+    rmse, rmse_cons = prediction_metrics.get_errors(y_data, onehot_data)
+    
+    assert np.sum(rmse) != 0.0, "should get nonzero errors for wrong predictions"
+    
+    
+    
+if __name__ == "__main__":
```
Comment (RandalJBarnes) [inline bivariate_normal/test_prediction_metrics.py:27]: Can you explain what this does, Dr. Barnes?

Comment (eabarnes1010) [reply bivariate_normal/test_prediction_metrics.py:27]: it only runs the code if it is called, not imported.

**Pull request merged**